alws/crud/build_node.py

```diff
@@ -175,8 +175,8 @@ def get_repo(repo_arch, is_debug):
         )
 
     if module_index:
-        results = await asyncio.gather(*[__get_rpm_package_info(
-            pulp_client, rpm.href) for rpm in rpms])
+        results = await asyncio.gather((__get_rpm_package_info(
+            pulp_client, rpm.href) for rpm in rpms))
```

Comment:
not sure what `gather` can unpack generator

Reply:
It actually unpacks it, tested locally